cinema/serializers.py

```diff
@@ -68,13 +81,67 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available"
+        )
+
+
+class TicketSerializer(serializers.ModelSerializer):
+    movie_session = MovieSessionListSerializer(read_only=True)
+
+    class Meta:
+        model = Ticket
+        fields = ("id", "row", "seat", "movie_session")
+
+    def validate(self, attrs):
+        data = super(TicketSerializer, self).validate(attrs)
+        Ticket.validate_ticket(
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
```

Comment:
In the `validate` method of `TicketSerializer`, accessing `attrs['movie_session']` without checking if it exists may lead to a `KeyError`. Consider using `attrs.get('movie_session')` and handling the case where it might be `None`.